docs/samples/python/span_marker_recognizer.py

```diff
@@ -0,0 +1,221 @@
+import logging
```

Comment:
Please add this sample to the list of samples https://github.com/microsoft/presidio/blob/main/docs/samples/index.md

Reply:
Hi @omri374 
Added the Span Marker sample in list of samples.